### PR TITLE
Add OSM and map.locate function and simple Header

### PR DESCRIPTION
**Pull request by Cathesso**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,8 +1,23 @@
+//import {Switch, Route} from "react-router-dom";
+import ExplorePage from "./pages/ExplorePage";
+import styled from "styled-components/macro";
+import "./css/App.css";
 
 function App() {
   return (
- <p>Don't Panic!</p>
```
Comment (yasminwaldeck) [inline frontend/src/App.js:4]: !!!!

File: frontend/src/css/App.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.App{
+    height: 10vh;
+    width: 100vw;
+    background-color: hotpink;
```
Comment (yasminwaldeck) [inline frontend/src/css/App.css:4]: das macht mir angst

**Review by yasminwaldeck — APPROVED** [review]
yay!

File: frontend/src/css/App.css (hunk 2 above)
Comment (Cathesso) [reply frontend/src/css/App.css:4]: Zu Recht :D

File: frontend/src/App.js [hunk 3]
```diff
@@ -1,8 +1,23 @@
+//import {Switch, Route} from "react-router-dom";
```
Comment (christophersiem) [inline frontend/src/App.js:1]:
```suggestion

```

File: frontend/src/components/StreetMap.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  useMapEvents,
+} from "react-leaflet";
+
+function MyComponent() {
+  const map = useMapEvents({
+    click: () => {
+      map.locate();
+    },
+    locationfound: (location) => {
+      console.log("location found:", location);
```
Comment (christophersiem) [inline frontend/src/components/StreetMap.js:15]:
```suggestion

```

File: frontend/src/css/Index.css [hunk 5]
```diff
@@ -0,0 +1,5 @@
+body{
+    margin: 0;
+    height: 100vh;
+    width: 100vw;
+}
```
Comment (christophersiem) [inline frontend/src/css/Index.css:5]: maybe use GlobalStyle from styled-components, so you don't have to mix it up with css files

**Review by fabianschmauder — COMMENTED** [review]
looks good. are you using npm or yarn? I would remove the package-lock or yarn-lock and use npm or yarn

File: frontend/src/App.js (hunk 3 above)
Comment (Cathesso) [reply frontend/src/App.js:1]: > looks good. are you using npm or yarn? I would remove the package-lock or yarn-lock and use npm or yarn

Gonna delete the yarn-lock :)

File: frontend/src/css/Index.css (hunk 5 above)
Comment (Cathesso) [reply frontend/src/css/Index.css:5]: I will give it a try :D

**Pull request merged**